src/drivers/driver_zephyr.h

```diff
@@ -200,7 +200,6 @@ struct zep_wpa_supp_dev_callbk_fns {
 			      union wpa_event_data *event);
 };
 
-#ifdef CONFIG_WIFI_NM_HOSTAPD_AP
```

Comment:
This a fixup and fixup commit aren't allowed in a PR, so, please use `fixup` and merge to the original commit,

Reply:
Updated.